app/models/helpers/helpers.rb

```diff
@@ -0,0 +1,13 @@
+module Helpers
```

Comment:
Can this go into app/models/concerns? 
Also, in the interest of keeping everything small, modular and single-responsibility, how about renaming this to e.g. 'date_validator.rb' instead of the generic 'helpers.rb' that may invite people to dump code here in the future?